project-base/tests/ShopBundle/Acceptance/acceptance/AdminProductSaveFormWithEmptyParametersCest.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Tests\ShopBundle\Acceptance\acceptance;
+
+use Tests\ShopBundle\Acceptance\acceptance\PageObject\Admin\LoginPage;
+use Tests\ShopBundle\Acceptance\acceptance\PageObject\Admin\ProductEditPage;
+use Tests\ShopBundle\Test\Codeception\AcceptanceTester;
+
+class AdminProductSaveFormWithEmptyParametersCest
+{
+    public function testSaveForm(
+        AcceptanceTester $me,
+        LoginPage $loginPage,
+        ProductEditPage $productEditPage
+    ) {
+        $me->wantTo('not able to save invalid form');
+        $loginPage->loginAsAdmin();
+
+        $productEditPage->saveInvalidForm(5);
+        $productEditPage->assertSaveFormErrorBoxVisible();
+
+        $productEditPage->cleanUpWorkspaceAfterChanges();
```

Comment:
why? could not you just call `$this->tester->acceptPopup()`?

Reply:
In this test the form is changed and browser will stop execution of next tests due to modal box that wants approval. Seems that opposite order of commands doesn't work if the test is ended and next test requests for different page modal box is not canceled.